settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,55 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: cloud
+              options:
+                  cloud:
+                      type: Value
```

Comment:
I assume this needs to point to a cloud file with params no?
I would say you need this as YamlFile type not "value".